JsonLogic/Rules/BooleanCastRule.cs

```diff
@@ -30,8 +37,6 @@ internal BooleanCastRule(Rule value)
 	/// <returns>The result of the rule.</returns>
 	public override JsonNode? Apply(JsonNode? data, JsonNode? contextData = null)
 	{
-		var value = Value.Apply(data, contextData);
```

Comment:
This wasn't being used.

Reply:
Oh, I probably had that in there as a debug so I could see what the result was.